bedrock/security/templates/security/bug-bounty/web-eligible-sites.html

```diff
@@ -254,4 +254,12 @@ <h3>VPN</h3>
     <li>vpn.mozilla.org</li>
   </ul>
 
+  <h3>Support</h3>
+  <ul class="mzp-u-list-styled">
+    <li>support.mozilla.org</li>
+  </ul>
+  <p>
+    <br> Please use our <a href="https://support.allizom.org">staging instance</a> for testing to avoid disrupting users.
```

Comment:
This `<br>` is adding an unnecessary blank line.
```suggestion
    Please use our <a href="https://support.allizom.org">staging instance</a> for testing to avoid disrupting users.
```